src/web/modules/gateway_accounts/filter.njk

```diff
@@ -106,6 +106,18 @@
       }}
     </div>
 
+    <div class="govuk-grid-column-one-quarter inputs-less-margin">
+      {{ booleanFilterSelect({
+          id: "switching_psp",
```

Comment:
other ids use snake case. unsure how important this is, but maintains consistency.